core/trino-server/src/main/provisio/trino.xml

```diff
@@ -68,6 +68,12 @@
         </artifact>
```

Comment:
Instead adding new plugin. I would prefer to document existing connector like PostgreSql.

Reply:
Document how? In the developer guide or in the connector docs?

Reply:
My goal is to show how much simpler a basic jdbc connector is compared to a regular connector. This example will work out of the box with any JDBC driver and the very basic data types, but it might be enough for lots of users to get started using Trino.